From: Boyuan Yang <byang@debian.org>
Date: Thu, 21 Oct 2021 18:37:04 -0400
Subject: doc: Use libjs-mathjax rather than cloudflare copy

---
 libs.doxy_template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs.doxy_template b/libs.doxy_template
index 34216b8..9cddfa0 100644
--- a/libs.doxy_template
+++ b/libs.doxy_template
@@ -1532,7 +1532,7 @@ MATHJAX_FORMAT         = HTML-CSS
 # The default value is: https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/.
 # This tag requires that the tag USE_MATHJAX is set to YES.
 
-MATHJAX_RELPATH        = https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/
+MATHJAX_RELPATH        = /usr/share/javascript/mathjax/
 
 # The MATHJAX_EXTENSIONS tag can be used to specify one or more MathJax
 # extension names that should be enabled during MathJax rendering. For example
