diff --git a/av1/common/alloccommon.c b/av1/common/alloccommon.c
--- a/av1/common/alloccommon.c
+++ b/av1/common/alloccommon.c
@@ -222,7 +222,7 @@
       // Calculate top and bottom line buffer size
       const int luma_stride =
           ALIGN_POWER_OF_TWO(cm->mi_params.mi_cols << MI_SIZE_LOG2, 4);
-      new_linebuf_size[plane] = sizeof(*cdef_info->linebuf) * num_bufs *
+      new_linebuf_size[plane] = sizeof(**cdef_info->linebuf) * num_bufs *
                                 (CDEF_VBORDER << 1) * (luma_stride >> shift);
       // Calculate column buffer size
       const int block_height =
